### DISCUSSION TITLE
=>websend  and authentication

### DISCUSSION THREAD

**Discussion by vegasville** [arendst/Tasmota#12187] [Q&A] [answered]
What is the syntax for =>websend commands to another Tasmota (Shelly1) device which has a password?
Thank you in advance!

Something like: 
#ifndef USE_SCRIPT
#define USE_SCRIPT
#endif
#define USE_BUTTON_EVENT

>D
>b
tmp=bt[1]
if tmp==1  
then
=>websend [192.168.1.3] cm?user=admin&password=powpow&cmnd=Power%20on
endif

**Comment by sfromis** [comment, marked as answer]
Explained [here](https://tasmota.github.io/docs/Commands/#websend). You should not build a URL, but provide the fields within [], and no query string keywords.

**Reply by vegasville** [reply]
So assuming the rest of the script is correct, example 1 suggests: 
=>websend [192.168.1.3:80,admin:powpow] POWER1 ON
...should work, I think

**Reply by sfromis** [reply]
Did you try it? As I'm not using scripting, I'm not saying anything for certain. You could try the WebSend command in the console first, to make sure that it is working, before trying to integrate it in the script.
